components/blocks/SiteFooter/LanguageSelector/LanguageSelector.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <select
+    v-model="selected"
+    class="language-selector"
+    role="listbox"
+    @change="onChange"
+  >
+    <option v-for="locale in locales" :key="locale.code" :value="locale.code">
+      {{ locale.name }}
+    </option>
+  </select>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api';
+import { Locale } from '@/types';
+
+export default defineComponent({
```

Comment:
Could you use the composition API instead of the Option API?

Reply:
Absolutely! I think that got prefilled by VSCode, and I just totally accepted the suggestion. I'll get that switched over to using the `@nuxtjs/composition-api` package.